controllers/notification.js

```diff
@@ -8,6 +8,15 @@ const getNotifications = async (req, res) => {
   res.status(200).json(notifications)
 }
 
+const clearNotifications = async (req,res) => {
+  const { id } = req.user
+
+  await notificationService.clearNotifications(id)
+
+  res.status(204).end()
+}

```

Comment:
Do we delete all notifications every time or we can delete certain notifications?
Or do we have another endpoint?
![image](https://github.com/ita-social-projects/SpaceToStudy-BackEnd/assets/83120263/4f76685d-bf9e-4b30-a54b-b6732d9a4316)

Reply:
currently we will have only for clearing all notifications, but yeap it's needed to add the task for deleting certain notification